generators/app/templates/src/redux/pushNotifications/actions.js

```diff
@@ -1,58 +1,46 @@
-import { stringArrayToObject } from '../../utils/arrayUtils';
+import { createTypes } from 'redux-recompose';
+
 import { isAndroid } from '../../constants/platform';
 import PushNotificationsService from '../../services/PushNotificationsService';
 
-export const actions = stringArrayToObject(
-  ['REGISTER', 'UPDATE_TOKEN', 'NOTIFICATION_RECEIVED'],
-  '@@PUSH_NOTIFICATIONS'
-);
+export const actions = createTypes(['REGISTER', 'NOTIFICATION_RECEIVED'], '@@PUSH_NOTIFICATIONS');
```

Comment:
`UPDATE_TOKEN` ?

Reply:
🤷‍♂️  it wasnt present in the reducer, so i removed it 👼